src/Microsoft.Diagnostics.DebugServices.Implementation/Module.cs

```diff
@@ -164,9 +186,51 @@ public virtual ImmutableArray<byte> BuildId
             }
         }
 
-        public abstract VersionData VersionData { get; }
+        public IEnumerable<PdbFileInfo> GetPdbFileInfos()
+        {
+            GetPEInfo();
+            Debug.Assert(_pdbFileInfos is not null);
+            return _pdbFileInfos;
+        }
 
-        public abstract string VersionString { get; }
+        public string GetSymbolFileName()
+        {
+            if (InitializeValue(Flags.InitializeSymbolFileName))
+            {
+                if (Target.OperatingSystem == OSPlatform.Linux)
```

Comment:
It should have a better name because it is meant only for Linux (and MacOS) symbol file name.  The PDB info property contains the file paths for Windows modules. There are usually multiple PDB records so if it clear which one to return.  

I really want to eventually breakdown the IModule interface into "generic" for all modules, "Windows" which would info the GetPdbInfoInfo(), Index* properties and the "unix" specific properties (BuildId) and methods (GetSymbolFileName()), That should make it clearer what this method returns.